cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
I added test for etcd2 -. etcd3 migration (by extending the existing test-update-objects test).
This should be good enough starting point.

[Rollback is not supported yet, so there is obviously no test for it.]